### ISSUE TITLE
Error sending status : check cmd/API with Apply for Service rule

### ISSUE THREAD

**Issue by PontiradioPR** [patrickpr/trapdirector#75]
Hello, I'm trying to collect traps that comes from some devices when a Digital Input change status (basically a "Start" and "Stop" event).
Actually I have configured and tested succesfully the Handler rules relatet do each DI:

![immagine](https://github.com/patrickpr/trapdirector/assets/137053752/34e8d338-6850-4512-8dad-5b3254f2df00)

Hovewer, since not each device has the same digital input labels, I have created a Host Custom variable for each digital input and assiged it to the service with the "apply for" rule in Icinga Director.
As soon I update my services with those rules, the Trapdirector returns to me the "Error sending status : check cmd/API" each time a trap is processed with a match.

Basically, I imagine that it is related to the fact that the service name with the apply for rule is not "DI1:" anymore but "DI1:[Host Variable]" so trapdirector send an incomplete variable to Icinga, but I am not able to find any solution to this issue, apart from creating specific services and also rules for each host (they are more than 100).

Do you have any solution or suggestion to share with me in order to pass the full service name from trapdirector to icinga2?

Very kind regards
